src/export/InstanceExporter.ts

```diff
@@ -650,3 +727,9 @@ export class InstanceExporter implements Fishable {
     return this.pkg;
   }
 }
+
+// This logic is extracted to its own function so we can easily test it
+// to ensure it exhibits the behavior we need.
+export function matchesPattern(obj: any, pattern: any): boolean {
+  return isMatch(obj, pattern);
+}
```

Comment:
I feel conflicted about having `matchesPattern` as a wrapper function for lodash's `isMatch` and the accompanying test cases. Since there's no additional logic in `matchesPattern`, these tests are against lodash's code, not ours. As you mentioned in the comment , you added the tests so that you're confident that `isMatch` is doing what you want it to do, and I also want to be confident of that. I recognize that the tests you have for `matchesPattern` could be rewritten to be tests that check for messages from `checkForNamelessSlices`, but that means rewriting a bunch of tests to be much longer. So, it's giving you more work to do, but it's also a test that more accurately targets SUSHI code, not an external library. Let me know what you think.

Reply:
Yeah, you're probably right. I originally did this because I thought the `matchesPattern` function was going to be _more_ than just lodash `isMatch`. When I discovered that it is, in fact, just `isMatch`, I had already written all the tests and it was 11pm the night before the sprint closed. So I opted for not re-doing all the tests and putting this message in instead.  But... you're probably right.

Reply:
Addressed in [44901a5](https://github.com/FHIR/sushi/pull/1200/commits/44901a59d50bf9007351b99ce146eda82e4cb322).